Creating an issue fails if inputs.issue_labels is not given

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
Creating an issue fails if inputs.issue_labels is not given

**What you expected to happen**:
Action is expected to create an issue without labels.

**How you configured .github/workflows/**:
```yaml
name: npm audit

on:
  schedule:
    - cron: '40 * * * *'

jobs:
  scan:
    name: npm audit
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      - name: install dependencies
        run: npm ci
      - uses: oke-py/npm-audit-action@v0.1.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          issue_assignees: oke-py
          issue_lables: vulnerability,test
```
This includes a typo issue_lables.

**Anything else we need to know?**:

Action returns an error below.
```
##[error]Validation Failed: {"resource":"Label","field":"name","code":"missing_field"}
##[error]Node run failed with exit code 1
```

--- a/.github/workflows/daily.yml
+++ b/.github/workflows/daily.yml
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v1
       - name: install dependencies
         run: npm ci
-      - uses: oke-py/npm-audit-action@v0.1.0
+      - uses: oke-py/npm-audit-action@v1.0.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           issue_assignees: oke-py

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ jobs:
       - uses: actions/checkout@v1
       - name: install dependencies
         run: npm ci
-      - uses: oke-py/npm-audit-action@v0.1.0
+      - uses: oke-py/npm-audit-action@v1.0.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           issue_assignees: oke-py

--- a/__tests__/issue.test.ts
+++ b/__tests__/issue.test.ts
@@ -8,8 +8,8 @@ describe('getIssueOption', () => {
     const expected: IssueOption = {
       title: 'npm audit found vulnerabilities',
       body: 'hi',
-      assignees: [''],
-      labels: ['']
+      assignees: undefined,
+      labels: undefined
     }
     expect(issue.getIssueOption('hi')).toEqual(expected)
   })

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "npm-audit-action",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "private": true,
   "description": "GitHub Action to run `npm audit`",
   "main": "lib/main.js",

--- a/src/issue.ts
+++ b/src/issue.ts
@@ -2,16 +2,20 @@ import * as core from '@actions/core'
 import {IssueOption} from './interface'
 
 export function getIssueOption(body: string): IssueOption {
+  let assignees
+  let labels
+
+  if (core.getInput('issue_assignees')) {
+    assignees = core.getInput('issue_assignees').replace(/\s+/g, '').split(',')
+  }
+  if (core.getInput('issue_labels')) {
+    labels = core.getInput('issue_labels').replace(/\s+/g, '').split(',')
+  }
+
   return {
     title: core.getInput('issue_title'),
     body,
-    assignees: core
-      .getInput('issue_assignees')
-      .replace(/\s+/g, '')
-      .split(','),
-    labels: core
-      .getInput('issue_labels')
-      .replace(/\s+/g, '')
-      .split(',')
+    assignees,
+    labels
   }
 }

--- a/src/main.ts
+++ b/src/main.ts
@@ -3,8 +3,8 @@ import * as github from '@actions/github'
 import stripAnsi from 'strip-ansi'
 import Octokit, {IssuesCreateResponse} from '@octokit/rest'
 import {Audit} from './audit'
-import * as issue from '../src/issue'
-import {IssueOption} from '../src/interface'
+import * as issue from './issue'
+import {IssueOption} from './interface'
 
 async function run(): Promise<void> {
   try {

